src/vpn-api/app.ts

```diff
@@ -19,50 +19,58 @@ import * as cluster from 'cluster';
 import * as _ from 'lodash';
 import * as os from 'os';
 
-import { logger, VERSION } from '../utils';
+import { getLogger, metricsServer, spawnChildren, VERSION } from '../utils';
 
 import { service } from './utils';
 import worker from './worker';
 
-['VPN_INSTANCE_COUNT']
+const logger = getLogger('vpn');
+
+[
+	'VPN_INSTANCE_COUNT',
+
+	'BALENA_API_HOST',
+	'VPN_SERVICE_API_KEY',
+	'VPN_HOST',
+
+	'VPN_BASE_SUBNET',
+	'VPN_BASE_PORT',
+	'VPN_BASE_MANAGEMENT_PORT',
+	'VPN_INSTANCE_SUBNET_BITMASK',
+]
 	.filter(key => process.env[key] == null)
 	.forEach((key, idx, keys) => {
-		logger.error(`${key} env variable is not set.`);
+		logger.emerg(`${key} env variable is not set.`);
 		if (idx === keys.length - 1) {
-			process.exit(1);
+			process.exitCode = 1;
 		}
 	});
 
 const VPN_INSTANCE_COUNT =
 	parseInt(process.env.VPN_INSTANCE_COUNT!, 10) || os.cpus().length;
+const VPN_VERBOSE_LOGS = process.env.VPN_VERBOSE_LOGS === 'true';
```

Comment:
Probably no reason to assign to this const since it's meant to be toggleable (and I don't think this var is used outside of initialising the toggle)